.github/workflows/ci.yml

```diff
@@ -25,9 +25,5 @@ jobs:
           key: ${{ runner.os }}-turbo-${{ github.sha }}
           restore-keys: |
             ${{ runner.os }}-turbo-
-      - run: yarn turbo run --filter='./packages/core*' lint
-      - run: yarn turbo run --filter='./packages/core*' test
-      - run: yarn turbo run --filter='./packages/node*' lint
-      - run: yarn turbo run --filter='./packages/node*' test
-      - run: yarn turbo run --filter='./packages/browser*' lint
-      - run: yarn turbo run --filter='./packages/browser*' test
+      - run: yarn turbo run lint --filter='@ht-sdks/events-sdk-js*'
+      - run: yarn turbo run test --filter='@ht-sdks/events-sdk-js*'
```

Comment:
decided to parallelize these to speed up the action and using the package name glob `events-sdk-js*` will auto-run for all our relevant packages